app/dashboard/templates/bounty/fulfill.html

```diff
@@ -93,6 +93,13 @@ <h4 class="text-center mb-4">{% trans "Submit Work" %}</h4>
                         {% endfor %}
                       </select>
                     </div>
+                    <div class="mt-2">
+                      <label class="form__label" for="videoDemoLink">{% trans "Video demo Link" %}</label>
+                      <input name='videoDemoLink' id='videoDemoLink' class="form__input font-body" type="text"
+                        placeholder="https://youtu.be/DJartWzDn0E" value=""
+                        required
```

Comment:
I think this should be required, not every bounty make sense to have a video

Reply:
Removed @octavioamu, this was required only for hackathon prizes (bounties)